eng/Dependencies.props

```diff
@@ -25,6 +25,8 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore" />
     <LatestPackageReference Include="Microsoft.CodeAnalysis.Razor" />
     <LatestPackageReference Include="Microsoft.CSharp" />
+    <LatestPackageReference Include="Microsoft.DotNet.HotReload.Agent" />
+    <LatestPackageReference Include="Microsoft.DotNet.HotReload.Agent.Data" />
```

Comment:
Is this essentially a sources package?

Reply:
Also, is the plan for this code to end up in the runtime eventually (To enable HotReload outside Blazor).